### PR TITLE
cls issue fixed

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
#Fixed CLS issue for wiki page

## Linked issues

 closes https://github.com/EveripediaNetwork/issues/issues/949

### CODE REVIEW

**Comment by kesar** [comment]
i think its still showing closed / without content which moves the content

**Comment by OleanjiKingCode** [comment]
> i think its still showing closed / without content which moves the content

Alright sir

**Comment by OleanjiKingCode** [comment]
After a discussion with Srujan we decided to use next middleware to solve this

**Comment by kesar** [comment]
> After a discussion with Srujan we decided to use next middleware to solve this

is there any PR about this?

**Comment by OleanjiKingCode** [comment]
> 

We later concluded to use normal css to for it

**Comment by kesar** [comment]
Captura de pantalla 2023-01-16 a las 10 09 00
Captura de pantalla 2023-01-16 a las 10 09 06
Captura de pantalla 2023-01-16 a las 10 09 09


cool, lets make sure we address this as soon as possible. 


@Royal-lobster plz also review if there are other stuff that might be interfering on the CLS / LCP

**Comment by kesar** [comment]
this pr has code commented please make sure pr is ready to review/merge

File: src/components/Admin/WikiEditorInsight/WikiEditorsInsight.tsx [hunk 1]
```diff
@@ -141,7 +141,7 @@ export const WikiEditorsInsightTable = () => {
       ipfs: string
       wikiId: string
     }[]
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    // eslint/no-unused-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (kesar) [inline src/components/Admin/WikiEditorInsight/WikiEditorsInsight.tsx:144]: this file hasnt changed so we wont need any new eslint rule

**Review by kesar — CHANGES_REQUESTED** [review]

**Comment by kesar** [comment]
can u provide a proof that the issue is solved? attach screenshots or some report

**Comment by OleanjiKingCode** [comment]
Before :

image


Now :  
image

that's what i use to judge the layout shift

**Comment by kesar** [comment]
> Before :
> 
> 
> 
> image
> 
> 
> 
> 
> 
> Now :  
> 
> image
> 
> 
> 
> that's what i use to judge the layout shift
> 
> 
> 
> 

nice one 👍 lets fix the comments and merge 🚀🚀

**Review by kesar — APPROVED** [review]

**Pull request merged**